MSAL/test/unit/MSALInteractiveRequestTests.m

```diff
@@ -533,11 +533,18 @@ - (void)testInteractiveRequestFlowWithUser
          XCTAssertEqualObjects(result.user.utid, @"1234-5678-90abcdefg");
          XCTAssertEqualObjects(result.user.name, [MSALTestIdTokenUtil defaultName]);
          XCTAssertEqualObjects(result.user.displayableId, [MSALTestIdTokenUtil defaultUsername]);
+         XCTAssertEqualObjects(result.user.environment, @"login.microsoftonline.com");
```

Comment:
these should really be separate tests, but we can address that afterwards as an overall sweep of the test suite.